content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -1,50 +1,62 @@
 ---
 title: Parallel Processing using Expansions
-content_template: templates/concept
+content_template: templates/task
 weight: 20
 ---
 
 {{% capture overview %}}
 
-In this example, we will run multiple Kubernetes Jobs created from
-a common template.  You may want to be familiar with the basic,
-non-parallel, use of [Jobs](/docs/concepts/workloads/controllers/jobs-run-to-completion/) first.
+This task demonstrates running multiple {{< glossary_tooltip text="Jobs" term_id="job" >}}
+based a common template. You can use this approach to process batches of work in
+parallel.
 
+For this example there are only three items: [_apple_, _banana_, _cherry_].
+The sample Jobs process each item simply by printing a string then pausing.
+
+See [using Jobs in real workloads](#using-jobs-in-real-workloads) to learn about how
+this pattern fits more realistic use cases.
 {{% /capture %}}
 
+{{% capture prerequisites %}}
 
-{{% capture body %}}
+You should be familiar with the basic,
+non-parallel, use of [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/).
 
-## Basic Template Expansion
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-First, download the following template of a job to a file called `job-tmpl.yaml`
+You will need the command-line utility `sed`, a working install of
+[Python](https://www.python.org/), and the Jinja2 template
+library for Python.
+If you have Python but not Jinja2, you can install Jinja2 by running:
+```shell
+pip install --user jinja2
+```
+{{% /capture %}}
 
-{{< codenew file="application/job/job-tmpl.yaml" >}}
 
-Unlike a *pod template*, our *job template* is not a Kubernetes API type.  It is just
-a yaml representation of a Job object that has some placeholders that need to be filled
-in before it can be used.  The `$ITEM` syntax is not meaningful to Kubernetes.
+{{% capture steps %}}
 
-In this example, the only processing the container does is to `echo` a string and sleep for a bit.
-In a real use case, the processing would be some substantial computation, such as rendering a frame
-of a movie, or processing a range of rows in a database.  The `$ITEM` parameter would specify for
-example, the frame number or the row range.
+## Create a template Job
 
```

Comment:
Possible edit:
```##Create a template for the Job```

Reply:
Template Job is more accurate in this context, IMHO.